server/remote/bitbucketserver/convert.go

```diff
@@ -51,6 +52,7 @@ func convertStatus(status model.StatusValue) string {
 // structure to the common Woodpecker repository structure.
 func convertRepo(from *internal.Repo) *model.Repo {
 	repo := model.Repo{
+		RemoteID:     strconv.FormatInt(int64(from.ID), 10),
```

Comment:
```suggestion
		RemoteID:     fmt.Sprint(from.ID),
```
isn't Sprint the new suggested way?